doc/http-api/client_swagger.yml

```diff
@@ -76,30 +85,188 @@ paths:
             type: array
             items:
               $ref: '#/definitions/RecvMessage'
+  /rooms:
+    post:
+      description: Creates a room.
+      parameters:
+        - in: body
+          name: room
+          required: true
+          schema:
+            $ref: '#/definitions/CreateRoomBody'
+      responses:
+        200:
+          description: The room was successfully created
+          schema:
+            $ref: '#/definitions/ResourceID'
+  /rooms/{id}:
+    get:
+      description: "Returns room's details"
+      parameters:
+        - in: path
+          name: id
+          required: true
+          type: string
+          description: The ID of a room
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RoomDetails'
+    put:
```

Comment:
Yes, that's good point. I think we'll go with the first approach, looks clearer.